app/containers/Dashboard/index.js

```diff
@@ -19,24 +19,104 @@ function formatBytes(bytes, decimals = 2) {
+const pieChartTitle = ['AP Vendors', 'Client Vendors'];
+
 const Dashboard = () => {
   const { customerId } = useContext(UserContext);
   const { loading, error, data } = useQuery(GET_ALL_STATUS, {
     variables: { customerId, statusDataTypes: ['CUSTOMER_DASHBOARD'] },
   });
+  const [toTime] = useState(
+    moment()
+      .valueOf()
+      .toString()
+  );
+  const [fromTime] = useState(
+    moment()
+      .subtract(24, 'days')
+      .valueOf()
+      .toString()
+  );
 
-  const titleList = ['Access Points', 'Client Devices', 'Usage Information'];
+  const {
+    loading: metricsLoading,
+    error: metricsError,
+    data: metricsData,
+    // refetch: metricsRefetch,
+  } = useQuery(FILTER_SYSTEM_EVENTS, {
+    variables: {
+      customerId,
+      fromTime,
+      toTime,
+      equipmentIds: [0],
+      dataTypes: ['StatusChangedEvent'],
+      limit: 100,
+    },
+  });
 
-  if (loading) {
-    return <Loading />;
-  }
+  const formatLineChartData = (list = []) => {
+    const lineChartData = {
+      inservicesAPs: { key: 'Inservice APs', value: [] },
+      clientDevices: [],
+      traffic: {
+        trafficBytesDownstream: { key: 'Down Stream', value: [] },
+        trafficBytesUpstream: { key: 'Up Stream', value: [] },
+      },
+    };
+    const clientDevicesKeyMap = { is2dot4GHz: '2.4GHz', is5GHzL: '5GHz (L)', is5GHzU: '5GHz (U)' };
+    list.forEach(
+      ({
+        eventTimestamp,
+        details: {
+          payload: {
+            details: {
+              equipmentInServiceCount,
+              associatedClientsCountPerRadio: radios,
+              trafficBytesDownstream,
+              trafficBytesUpstream,
+            },
+          },
+        },
+      }) => {
+        lineChartData.inservicesAPs.value.push([eventTimestamp, equipmentInServiceCount]);
+        Object.keys(radios).forEach(key => {
+          if (lineChartData.clientDevices.length) {
+            const index = lineChartData.clientDevices.findIndex(
+              item => item.key === clientDevicesKeyMap[key]
+            );
+            if (index >= 0) {
+              lineChartData.clientDevices[index].value.push(radios[key]);
+              return;
+            }
+          }
+          lineChartData.clientDevices.push({
+            key: clientDevicesKeyMap[key],
+            value: [radios[key]],
+          });
+        });
+        lineChartData.traffic.trafficBytesDownstream.value.push([
+          eventTimestamp,
+          trafficBytesDownstream,
+        ]);
+        lineChartData.traffic.trafficBytesUpstream.value.push([
+          eventTimestamp,
+          trafficBytesUpstream,
+        ]);
+      }
+    );
+
+    return lineChartData;
+  };
 
-  if (error) {
-    return <Alert message="Error" description="Failed to load Dashboard" type="error" showIcon />;
-  }
+  const lineChartsData = useMemo(
+    () => formatLineChartData(metricsData?.filterSystemEvents?.items),
+    [metricsData]
+  );
 
   const status =
```

Comment:
put this and everything below inside the useMemos. It's all part of the calculations